delorean-user-data.txt

```diff
@@ -280,18 +280,19 @@ write_files:
     permissions: 0755
 
 bootcmd:
-#  - echo "" >> /etc/ssh/sshd_config
-#  - echo "Port 330" >> /etc/ssh/sshd_config
-#  - systemctl restart sshd
+  - echo "" >> /etc/ssh/sshd_config
+  - echo "Port 3300" >> /etc/ssh/sshd_config
+  - echo "Port 22" >> /etc/ssh/sshd_config
```

Comment:
Port 22 was actually just a back to avoid lock out, I'd like to avoid it.
@javierpena commented out those lines, looks like sshd changes are not reliably applied within client-init script and, indeed, at first I was able to login only over 22 to the new instance...TBI
I've since removed 22 from security group so only publicly available port is 3300 and 22 is a backup in case something goes wrong with sshd config.